src/component/AttendingEvents.js

```diff
@@ -0,0 +1,103 @@
+import { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { Button, Image } from 'react-bootstrap';
+import { BiLeftArrow, BiRightArrow } from 'react-icons/bi';
+import { attendingEvents } from '../redux/event/event';
+import SelectEvent from './SelectEvent';
+import '../css/AttendingEvents.css';
+
+const AttendingEvents = () => {
+  const dispatch = useDispatch();
+  const attendingEventsAction = bindActionCreators(attendingEvents, dispatch);
+  const events = useSelector((state) => state.events.attendingEvents);
+  const loading = useSelector((state) => state.events.loading);
+
+  const cardsPerPage = 3;
+  const lastPage = Math.ceil(events.length / cardsPerPage);
+
+  const [activePage, setActivePage] = useState(1);
+
+  useEffect(() => {
+    setActivePage(1);
+  }, [events]);
+
+  useEffect(async () => {
+    if (loading) {
+      await attendingEventsAction();
+    }
+  }, []);
+
+  const limitLength = (str) => {
+    if (str.length > 20) {
+      return `${str.slice(0, 20)}...`;
+    }
+    return str;
+  };
+
+  const renderEvents = () => {
+    const result = [];
+    for (
+      let i = (activePage - 1) * cardsPerPage;
+      i < Math.min(cardsPerPage * activePage, events.length);
+      i += 1
+    ) {
+      let date = new Date(events[i].date);
+      const dd = String(date.getDate()).padStart(2, '0');
+      const mm = String(date.getMonth() + 1).padStart(2, '0');
+      const yyyy = date.getFullYear();
+
+      date = `${mm}/${dd}/${yyyy}`;
+
+      result.push(
+        <div
+          className="d-flex flex-column align-items-center justify-content-center mx-2"
+          key={events[i].id}
+          style={{ width: '18rem' }}
```

Comment:
- [ ] Remember that using the `style` attribute is still considered a bad practice, you should be able to use a class to personalize this.